CMake/resolve_dependency_modules/gflags.cmake

```diff
@@ -28,24 +28,25 @@ FetchContent_Declare(
   gflags
   URL ${VELOX_GFLAGS_SOURCE_URL}
   URL_HASH ${VELOX_GFLAGS_BUILD_SHA256_CHECKSUM}
-  PATCH_COMMAND git apply ${CMAKE_CURRENT_LIST_DIR}/gflags/gflags-config.patch)
+  PATCH_COMMAND git apply ${CMAKE_CURRENT_LIST_DIR}/gflags/gflags-config.patch
+                OVERRIDE_FIND_PACKAGE EXCLUDE_FROM_ALL SYSTEM)
 
-set(GFLAGS_BUILD_STATIC_LIBS ON)
-set(GFLAGS_BUILD_gflags_LIB ON)
-set(GFLAGS_BUILD_gflags_nothreads_LIB ON)
-set(GFLAGS_IS_SUBPROJECT ON)
 # glog relies on the old `google` namespace
 set(GFLAGS_NAMESPACE "google;gflags")
```

Comment:
nit: just noiced this is the ; intentional ?

Reply:
Yes, that's how you create a list in CMake :)